### PR TITLE
Update readme- First cut

### PR DESCRIPTION
**Pull request by jjw24**
@NickSeagull @SysC0mp @Kalaschnik @theClueless First cut of the readme update. Please let me know your thoughts, what I should add/remove/update.

Cheers

https://github.com/Flow-Launcher/Flow.Launcher/issues/3

### CODE REVIEW

**Comment by SysC0mp** [comment]
Looking great so far. I think that is something that will grow and being edited on and on while we work on the launcher. We can for example add another section for the plugins later on and how to create & integrate them.

File: README.md [hunk 1]
```diff
@@ -1,89 +1,75 @@
...
+[![Github All Releases](https://img.shields.io/github/downloads/Flow-Launcher/Flow.Launcher/total.svg)](https://github.com/Flow-Launcher/Flow.Launcher/releases)
+[![GitHub release (latest by date)](https://img.shields.io/github/v/release/Flow-Launcher/Flow.Launcher)](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest)
+![GitHub Release Date](https://img.shields.io/github/release-date/Flow-Launcher/Flow.Launcher)
+[![Build Status](https://dev.azure.com/Flow-Launcher/Flow.Launcher/_apis/build/status/Flow.Launcher?branchName=master)](https://dev.azure.com/Flow-Launcher/Flow.Launcher/_build/latest?definitionId=1&branchName=master)
 
-**WoX** is a launcher for Windows that simply works. It's an alternative to [Alfred](https://www.alfredapp.com/) and [Launchy](http://www.launchy.net/). You can call it Windows omni-eXecutor if you want a long name.
+Nickname flow, full name Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
```
Comment (NickSeagull) [inline README.md:10]: I'd make this sentence simpler:

```suggestion
Flow Launcher. Dedicated to make your workflow flow more seamlessly. Aimed at being more than an app launcher, it searches, integrates and expands on functionalities. Flow will continue to evolve, designed to be open and built with the community at heart.
```

File: README.md [hunk 2]
```diff
@@ -1,89 +1,75 @@
...
-------------
+- Search everything from applications, folders, bookmarks, YouTube, Twitter and more. All from the comfort of your keyboard without ever touching the mouse.
+- Run batch and PowerShell commands as Administrator or a different user.
+- Support languages from Chinese to Italian and more.
+- Support of wide range of plugins.
+- Fully portable.
 
-View new features released from this fork since Flow.Launcher v1.3.524: [new releases](https://github.com/jjw24/Flow.Launcher/releases)
+## Running Flow Launcher
+Remember to star it, flow will love you more :)
```
Comment (NickSeagull) [inline README.md:27]: I'd remove this one 

```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,89 +1,75 @@
...
 
-To install this fork's version of Flow.Launcher, you can **download** it [here](https://github.com/jjw24/Flow.Launcher/releases/latest).
+| [Windows 7 and up](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest)
+| ------------- |
 
-To install the upstream version:
+Windows may complain about security due to code not being signed. This is fine, you can proceed to install it.
```
Comment (NickSeagull) [inline README.md:32]: Perhaps add an explanation about signed packages?

File: README.md (hunk 1 above)
Comment (jjw24) [reply README.md:10]: Haha I was trying to sound a bit playful with it, not feeling it?

Also that we can refer to it as Flow for short if we want to later on.

File: README.md (hunk 2 above)
Comment (jjw24) [reply README.md:27]: Don't want to remind users? Thought it's a good idea

File: README.md (hunk 1 above)
Comment (NickSeagull) [reply README.md]: At first I liked it, but I think that sounding impersonal might give it a better feel :)

File: README.md (hunk 2 above)
Comment (NickSeagull) [reply README.md]: Yeah, but perhaps near the top or the bottom, writing this under the Running section feels weird

File: README.md [hunk 4]
```diff
@@ -1,89 +1,72 @@
...
-- Highlighting of how results are matched during query search
-- Open web search result as a tab or a new window
-- Automatic update
-- Reload/update plugin data
+| [Windows 7 and up](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest)
+| ------------- |
 
-Installation
-------------
+Windows may complain about security due to code not being signed, this will be completed at a later stage. If you downloaded from this repo, you are good to continue the set up.
```
Comment (NickSeagull) [inline README.md:25]: 🎉

File: README.md (hunk 2 above)
Comment (jjw24) [reply README.md]: What about this at the end of the Running section:
"Remember to start it, you will get more Chi from Flow :)"

Comment (NickSeagull) [reply README.md]: I liked the original message, but I'd put it right before the `features` section. I think that's a nice place 😄

Comment (jjw24) [reply README.md]: Done :)

**Pull request merged**